src/pybind/mgr/cephadm/module.py

```diff
@@ -1252,6 +1277,24 @@ def _refresh_host_devices(self, host):
         self.cache.save_host(host)
         return None
 
+    def _deploy_etc_ceph_ceph_conf(self, host: str) -> Optional[str]:
+        try:
+            with self._remote_connection(host) as tpl:
+                conn, connr = tpl
+                out, err, code = remoto.process.check(
+                    conn,
+                    ['mkdir', '-p', '/etc/ceph'])
```

Comment:
what if a different output dir was used during bootstrap (e.g. vstart)?
```
$ cephadm bootstrap ... --output-dir ~/ceph/build/
```

Reply:
Under no circumstance, I'd automatically enable `manage_etc_ceph_ceph_conf` if `output-dir` is something different than `/etc/ceph`